src/components/Products/ProductArticle.js

```diff
@@ -0,0 +1,102 @@
+import { Card, Icon, Image, Text } from "../Reusable"
+
+const ProductArticle = ({ prd }) => {
+    const {
+
+        name,
+        company,
+        size,
+        price,
+        offerPrice,
+        img: {
+            srcSet,
+            alt,
+            sizes
+        },
+        rating,
+        quantity
+    } = prd
+
+    return (
+        <Card classes="pd-xs pos-relative">
+            {
+                offerPrice &&
+                <Text classes="pos-absolute tr-1 bg-warn txt-md txt-600 txt-primary txt-ucase pd-xs">offer</Text>
+            }
+            {
+                quantity < 15 && <Text classes='txt-cap pos-absolute tl-0 bg-err txt-md'>{`only ${quantity} left.`}</Text>
+            }
+
+            <Image srcSet={srcSet} alt={alt} sizes={sizes} />
+
+            <Text classes="txt-md txt-cap txt-primary">{company}</Text>
+
+            <Text classes="txt-md txt-cap txt-primary mg-btm-xs">{name}</Text>
+
+            <div className="flx mg-btm-xs">
+                <Text classes='txt-md txt-primary txt-cap mg-right-s'>sizes</Text>
+                {
+                    size.map(size => <Text key={size} classes='txt-md txt-primary mg-right-xs'>{size}</Text>)
+                }
+            </div>
+
+            <div className="flx">
+                {
+                    offerPrice ?
+                        <div className="flx flx-column">
+
+                            <div className="flx">
+
+                                <Text
+                                    classes="txt-cut txt-md txt-cap txt-off-secondary">rs.
+                                    {price}
+                                </Text>
+
+                                <Text classes="txt-md txt-cap txt-primary mg-right-xs">
+                                    rs. {offerPrice}
+                                </Text>
+
+                            </div>
+
+                            <Text classes="txt-md txt-cap txt-success">
+                                {Math.round((price - offerPrice) / price * 100)}% off
+                            </Text>
+
+                        </div> :
+
+                        <Text classes="txt-md txt-cap txt-primary mg-btm-xs">
+                            rs. {price}
+                        </Text>
+                }
+
+            </div>
+
+            <div className='dis-inblock'>
+
+                <Icon classes={`${rating >= 1 ? 'txt-warn' : 'txt-off-secondary'}`}>
+                    star
+                </Icon>
+
+                <Icon classes={`${rating >= 2 ? 'txt-warn' : 'txt-off-secondary'}`}>
+                    star
+                </Icon>
+
+                <Icon classes={`${rating >= 3 ? 'txt-warn' : 'txt-off-secondary'}`} >
+                    star
+                </Icon>
+
+                <Icon classes={`${rating >= 4 ? 'txt-warn' : 'txt-off-secondary'}`} >
+                    star
+                </Icon>
```

Comment:
You could use forloop and then map it in jsx, so in this repetitive code is avoided. Let me know if you are not clear with this.

Reply:
yes @ShekharDhangar. i will be adding the logic later. this is just for ui purpose. thank you.